clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/validators/StandardKubernetesAttributeValidator.groovy

```diff
@@ -26,6 +26,7 @@ class StandardKubernetesAttributeValidator {
   static final prefixPattern = /^[a-z0-9]+$/
   static final cpuPattern = /^(\d+m)?$/
   static final memoryPattern = /^(\d+(Mi|Gi))?$/
+  static final serverGroupNamePattern = /^[a-zA-Z0-9]+(-[a-zA-Z0-9]+)+$/
```

Comment:
As discussed offline, I think we can drop the base server group name validation. If it's not found, it's not found. Don't see why we have to hold an opinion about the format of the base server group's name.